backend/app/usecase/match/contains_all.go

```diff
@@ -0,0 +1,19 @@
+package match
+
+import "strings"
+
+var _ Keyword = (*ContainsAll)(nil)
+
+// ContainsAll checks if an input contains all the elements of a list.
+type ContainsAll struct {
+}
+
+// IsMatch checks if the input contains all the words.
+func (c ContainsAll) IsMatch(words []string, input string) bool {
+	for _, word := range words {
+		if !strings.Contains(input, word) {
+			return false
+		}
+	}
+	return true
+}
```

Comment:
It seems that this doesn't have any dependencies. Can this be a standalone function instead?

Reply:
I agree!

Reply:
Please remove the structs and just export the functions!